extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/Functions.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.plugin.spark.authz.gen
+
+import org.apache.kyuubi.plugin.spark.authz.serde.{FunctionDesc, FunctionNameFunctionTypeExtractor, FunctionSpec, FunctionTypeDesc, QualifiedNameStringFunctionExtractor}
+import org.apache.kyuubi.plugin.spark.authz.serde.FunctionType._
+
+object Functions {
+
+  val clazz = classOf[QualifiedNameStringFunctionExtractor]
+
+  val functionTypeDesc = {
```

Comment:
The variable name is too common to reflect FunctionTypeDesc fetching from the name field. Also this variable is unnecessary as it is never reused.